mods/email/web/style.css

```diff
@@ -645,3 +645,25 @@ hr {
     max-width: none;
   }
 }
+
+#settings-dropdown {
+  width: 400 px;
```

Comment:
Has this been formatted with i.e. prettier?  
Even if this works, it looks confusing and is different from all the rest not according to any convention. I.e. if a dev wants to find a replace values this can create issues:  
```css
width: 400 px;
```
Better write it correct:
```css
width: 400px;
```

This applies to all values

Reply:
Fixed, I think that was a prettier thing because most of that data was a copy/paste from another file to make css more local.